Get_all_brand.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import pytest
+
+API_URL = 'https://automationexercise.com/api/brandsList'
+
+
+def test_api_status():
+    response = requests.get(API_URL)
+    assert response.status_code == 200
+
+
+def test_api_content():
+    response = requests.get(API_URL)
+    assert response.headers['Content-Type'] == 'application/json'
```

Comment:
verify the content in the response

Reply:
@amitsahu8 Changes made. Kindly review it.